src/main/java/uk/gov/justice/digital/delius/jpa/filters/ContactFilter.java

```diff
@@ -100,6 +113,20 @@ public Predicate toPredicate(Root<Contact> root, CriteriaQuery<?> query, Criteri
         return cb.and(predicates.toArray(new Predicate[0]));
     }
 
+    private void nonRarActivityOnlyFilter(Root<Contact> root, CriteriaBuilder cb, ImmutableList.Builder<Predicate> predicateBuilder) {
+        predicateBuilder.add(cb.equal(root.get("rarActivity"), "N"));
+        predicateBuilder.add(cb.or(cb.isNull(root.get("requirement")),
```

Comment:
are you missing the `softDeleted` filter?

Reply:
Updated the filter